app/controllers/api/comments_controller.rb

```diff
@@ -12,6 +12,16 @@ def create
     end
   end
 
+  def index
+    if current_debate
+      comments = GetComments.new(current_debate.id).to_h
+
+      render json: comments, status: 200
+    else
+      render json: { error: 'Debate not found' }, status: :not_found
```

Comment:
W tej metodzie powtórzyło się (i teraz jest w obu metodach):

`if current_debate`
..`...`
`else`
..`render json: { error: 'Debate not found' }, status: :not_found`
`end`

Można to wyekstraktować (może nawet do `application_controller` skoro mamy tam metodę `current_debate`), na przykład:

`before_action :require_current_debate`

`def require_current_debate`
..`unless current_debate`
....`render json: { error: 'Debate not found' }, status: :not_found`
..`end`
`end`